Update mcp9808_simpletest.py

signalmeister opened the pull request:
The original example code did not show the syntax for setting up a module with anything other than the default address. While this information is to be found in the object definition document, a simple amendment to the sample code gives the default and an amended address example.

signalmeister committed:
Update mcp9808_simpletest.py

The original example code did not show the syntax for setting up a module with anything other than the default address. While this information is to be found in the object definition document, a simple amendment to the sample code gives the default and an amended address example.

siddacious reviewed (approved):
Perfect! Well documented, thanks!

siddacious commented:
Heya, the code here is perfect but the PR needs to be made in the adafruit/Adafruit_CircuitPython_MCP9808 repo, not against your fork. Easy mistake!

Go ahead and go to this page,
https://github.com/adafruit/Adafruit_CircuitPython_MCP9808/compare/

 select the "compare across forks" button, select your fork and select the branch on your fork that the changes are on **for the "compare" fields**
**
![image](https://user-images.githubusercontent.com/1183985/65628048-836b4300-df85-11e9-920f-4ba7b146579b.png)

Let me know if you have any questions! Almost there ;)

signalmeister commented:
I think I followed that and have done what was needed!

> On 25 Sep 2019, at 19:16, siddacious <notifications@github.com> wrote:
> 
> Heya, the code here is perfect but the PR needs to be made in the adafruit/Adafruit_CircuitPython_MCP9808 repo, not against your fork. Easy mistake!
> 
> Go ahead and go to this page, select the "compare across forks" button, select your fork and select the branch on your fork that the changes are on for the "compare" fields
> **
>  <https://user-images.githubusercontent.com/1183985/65628048-836b4300-df85-11e9-920f-4ba7b146579b.png>
> Let me know if you have any questions! Almost there ;)
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/signalmeister/Adafruit_CircuitPython_MCP9808/pull/1?email_source=notifications&email_token=AK5IOOB3ZZ6FDFM34UAQKC3QLOTIZA5CNFSM4I2EMXL2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD7S3EIA#issuecomment-535147040>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AK5IOOGXPXVY3YACLELF6SLQLOTIZANCNFSM4I2EMXLQ>.
>

signalmeister committed:
Update mcp9808_simpletest.py

I am not really sure that I understood your comments being a total novice at this process - I do not know what a lint check being unhappy means.
If there are still issues, can you please let me know and I hope I can understand your special terms.

kattni committed:
lint, update comments

kattni committed:
address change request
